helm/fiaas-skipper/values.yaml

```diff
@@ -38,3 +38,19 @@ autoUpdate: true
 service:
   type: ClusterIP
 fiaas_override_yaml: null
+fiaasDeployDaemonConfigMaps: {}
+# example of how to use fiaasDeployDaemonConfigMaps:
+# (can not be used at the same time as `addFiaasDeployDaemonConfigmap`)
+#fiaasDeployDaemonConfigMaps:
+#  dev:
```

Comment:
I didn't realize that these were namespaces at first. Since this is just an example, maybe change them to something like "namespace-foo", "namespace-bar", to make it clearer?

Reply:
Do you just mean that the example was confusing, or do you think it would be useful to change the actual value structure? Something like this might be a bit more explicit: 
```
fiaasDeployDaemonConfigMaps:
  namespaces:
    dev:
      ...
    prod:
      ...
```

Reply:
I was actually just thinking that the example was confusing, but when you mention it, your suggestion has merit too.

I'm satisfied if you just change the example, but if you think being explicit in the structure is worth the effort to implement I'm all for that.

Reply:
I'll change the structure to match my suggestion above, it should be easy to do